Add google_bigquery_table to version 5 upgrade doc for #8460 (#8611)
GoogleCloudPlatform/magic-modules: mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown

The provider will now enforce at plan time that at most one of these fields be set.

### `schema` can only be represented as a JSON array with non-null elements.

> The provider will now enforce at plan time that `schema` can be unmarshalled into a valid JSON array with non-null elements.

Reviewer: "unmarshalled" is a bit Go-specific - what would you think about this language instead?

```suggestion
The provider will now enforce at plan time that `schema` is a valid JSON array with non-null elements.
```

Author: Makes sense, thanks, updated.
